dependency-check-suppressions.xml

```diff
@@ -66,6 +66,38 @@
         <notes>Temporary suppression to ignore CVE with com.google.guava. See RDM-3372</notes>
         <cve>CVE-2018-10237</cve>
     </suppress>
+    <suppress>
+        <notes>Temporary suppression</notes>
+        <cve>CVE-2018-1000873</cve>
+    </suppress>
+    <suppress>
+        <notes>Temporary suppression</notes>
+        <cve>CVE-2018-14718</cve>
+    </suppress>
+    <suppress>
+        <notes>Temporary suppression</notes>
+        <cve>CVE-2018-14719</cve>
+    </suppress>
+    <suppress>
+        <notes>Temporary suppression</notes>
+        <cve>CVE-2018-14720</cve>
+    </suppress>
+    <suppress>
+        <notes>Temporary suppression</notes>
+        <cve>CVE-2018-14721</cve>
+    </suppress>
+    <suppress>
+        <notes>Temporary suppression</notes>
+        <cve>CVE-2018-19360</cve>
+    </suppress>
+    <suppress>
+        <notes>Temporary suppression</notes>
+        <cve>CVE-2018-19361</cve>
+    </suppress>
+    <suppress>
+        <notes>Temporary suppression</notes>
+        <cve>CVE-2018-19362</cve>
+    </suppress>
```

Comment:
Could we exclude the suppressions from this PR so that they are properly addressed independently?